en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml

```diff
@@ -3444,226 +3554,173 @@ paths:
                 type: string
             Location:
               description: |
-                The URL of the uploaded mediation policy of the API.
+                The URL of the newly created resource.
               schema:
                 type: string
             Content-Type:
               description: |
                 The content type of the body.
               schema:
                 type: string
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Mediation'
+          content: {}
         400:
           $ref: '#/components/responses/BadRequest'
+        403:
+          $ref: '#/components/responses/Forbidden'
         404:
           $ref: '#/components/responses/NotFound'
         412:
           $ref: '#/components/responses/PreconditionFailed'
       security:
         - OAuth2Security:
             - apim:api_create
-            - apim:mediation_policy_create
+            - apim:api_manage
       x-code-samples:
         - lang: Curl
-          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          -H "Content-Type: multipart/form-data" -F mediationPolicyFile=@TokenExchange.xml
-          -F type=in "https://127.0.0.1:9443/api/am/publisher/v2/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/mediation-policies"'
+          source: 'curl -k -X PUT -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          -F file=@api.wsdl "https://127.0.0.1:9443/api/am/publisher/v3/apis/96077508-fd01-4fae-bc64-5de0e2baf43c/wsdl"'
 
-  /apis/{apiId}/mediation-policies/{mediationPolicyId}:
+  /apis/{apiId}/graphql-policies/complexity:
     get:
       tags:
-        - API Mediation Policy
-      summary: Get an API Specific Mediation Policy
+        - GraphQL Policies
+      summary: Get the Complexity Related Details of an API
       description: |
-        This operation can be used to retrieve a particular API specific mediation policy.
-      operationId: getAPIMediationPolicyByPolicyId
+        This operation can be used to retrieve complexity related details belonging to an API by providing the API id.
```

Comment:
Should be "complexity-related details"
Also, "API ID". Kindly check for "ID" in other descriptions as well.

Reply:
Fixed.